server/src/mailing/mx_send_mail.c

```diff
@@ -0,0 +1,95 @@
+#include <stdio.h>
+#include <string.h>
+#include <curl/curl.h>
+
+/* This is a simple example showing how to send mime mail using libcurl's SMTP
+ * capabilities. For an example of using the multi interface please see
+ * smtp-multi.c.
+ *
+ * Note that this example requires libcurl 7.56.0 or above.
+ */
+ #define USERNAME "uchat.llc@gmail.com"
+ #define PASSWORD "Qwerty1357"
+
+ #define FROM    "uchat.llc@gmail.com"
+ #define TO      "5810976@stud.nau.edu.ua"
+ #define CC      "uchat.llc@gmail.com"
+
+
+ static void smtp_registation(CURL *curl) {
+     /* Set username and password */
+     curl_easy_setopt(curl, CURLOPT_USERNAME, USERNAME);
+     curl_easy_setopt(curl, CURLOPT_PASSWORD, PASSWORD);
+     curl_easy_setopt(curl, CURLOPT_URL, "smtps://smtp.gmail.com:465");
+ }
+
+ static void setup_email_recipients(CURL *curl, struct curl_slist *recipients,
+                                    char *to) {
+     curl_easy_setopt(curl, CURLOPT_MAIL_FROM, FROM);
+     recipients = curl_slist_append(recipients, to);
+     recipients = curl_slist_append(recipients, CC);
+     curl_easy_setopt(curl, CURLOPT_MAIL_RCPT, recipients);
+ }
+
+ static void setup_email_header(CURL *curl, struct curl_slist *headers,
+                                char *to) {
+     const char **line;
+     char to_user[1024];
+     sprintf(to_user, "To: %s", to);
+     const char *headers_text[] = {
+         "Date: Tue, 22 Aug 2017 14:08:43 +0100",
```

Comment:
Hmm...

Reply:
I have already changed this code-style mistake but forget to push new version